board/views.py

```diff
@@ -1,36 +1,82 @@
+import datetime
+import requests
 from django.shortcuts import render
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
-import datetime
+
+
+igdb_url = 'https://api.igdb.com/v4/'
+igdb_headers = {'Client-ID': 'so25gzj451xajgpll19wm0badrh9sk',
+                'Authorization': 'Bearer gd0r59g1nm5rem9j0bcj4ssmtrg4l8'}
+
+
+def get_img_url(image_id, size='screenshot_big'):
+    return 'https://images.igdb.com/igdb/image/upload/t_{}/{}.jpg'.format(size, image_id)
 
 
 class Game:
-    name = 'PACMAN'
-    desc = 'arcade video game'
-    img_url = 'static/board/img/pacman.jpg'
-    desc_full = '''Pac-Man is a maze arcade game developed and released by Namco in 1980.
-     The original Japanese title of Puck Man was changed to Pac-Man for international releases as a preventative 
-     measure against defacement of the arcade machines by changing the P to an F. Outside Japan, the game was published
-     by Midway Games as part of its licensing agreement with Namco America. The player controls Pac-Man, who must eat 
-     all the dots inside an enclosed maze while avoiding four colored ghosts. Eating large flashing dots called 
-     "Power Pellets" causes the ghosts to turn blue, allowing Pac-Man to eat them for bonus points. 
-     '''
-    release = 'Nov 1992'
-    screen_url = ['../../static/board/img/screen1.png', '../../static/board/img/screen2.jpg',
-                 '../../static/board/img/screen3.png',
-                 '../../static/board/img/screen1.png', '../../static/board/img/screen2.jpg',
-                 '../../static/board/img/screen3.png']
-    genres = ['Arcade', 'Action']
-    platforms = ['PC', 'PS4']
-    rating = {'users': [7.32, 123], 'critics': [7.11, 123]}
-    id = 1
+    def __init__(self, game_id):
+        params = {
+            'fields': 'id, cover.image_id, name, '
+                      'summary, screenshots.image_id, '
+                      'genres.name, release_dates, '
+                      'platforms.name, '
+                      'aggregated_rating, aggregated_rating_count, '
+                      'rating, rating_count ',
+            'filter[id][eq]': game_id
+        }
+
+        res = requests.post(igdb_url + 'games/', headers=igdb_headers, params=params).json()[0]
+
+        self.id = game_id
+        self.name = res['name']
+        self.desc = ' '.join([genre['name'] for genre in res['genres']])
+        self.desc_full = res['summary']
```

Comment:
Try to avoid abbreviations in namings, `full_description` is not much longer, but more readable.